tools/sync-npm-deps-to-tsc-projects.ts

```diff
@@ -40,6 +40,12 @@ const main = async (argv: string[], env: NodeJS.ProcessEnv) => {
   const globbyOptions: GlobbyOptions = {
     cwd: PROJECT_DIR,
     absolute: true,
+    ignore: [
```

Comment:
@ruzell22 Please place a direct link to the follow-up issue in a comment here in the code so that whoever comes around and looks at these hardcoded paths will know that there is a task in progress to remediate the problem.